ci/github-script/prepare.js

```diff
@@ -24,6 +25,32 @@ module.exports = async ({ github, context, core }) => {
 
     const { base, head } = prInfo
 
+    const baseClassification = classify(base.ref)
+    core.setOutput('base', baseClassification)
+    console.log('base classification:', baseClassification)
+
+    const headClassification =
+      base.repo.full_name === head.repo.full_name
+        ? classify(head.ref)
+        : // PRs from forks are always considered WIP.
+          { type: ['wip'] }
+    core.setOutput('head', headClassification)
+    console.log('head classification:', headClassification)
+
+    if (baseClassification.type.includes('channel')) {
+      const { stable, version } = baseClassification
+      const correctBranch = stable ? `release-${version}` : 'master'
```

Comment:
I guess at this early stage we _can't_ know whether it should be release/master or staging.